src/screens/OrganizationVenues/OrganizationVenues.spec.tsx

```
@@ -282,177 +49,132 @@ const renderOrganizationVenue = (link: ApolloLink): RenderResult => {
+
     await waitFor(() => {
-      expect(screen.getByTestId('venue-item1')).toHaveTextContent(
-        /Updated Venue 1/i,
-      );
-      expect(screen.getByTestId('venue-item2')).toHaveTextContent(
-        /Updated Venue 2/i,
+      const venueCards = screen.getAllByTestId((testId) =>
+        testId.startsWith('capacity-'),
       );
+      const capacities = venueCards.map((card) => {
+        return parseInt(
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)